Source/WebCore/loader/cache/CachedCSSStyleSheet.cpp

```diff
@@ -139,11 +139,18 @@ bool CachedCSSStyleSheet::mimeTypeAllowedByNosniff() const
     return parseContentTypeOptionsHeader(response().httpHeaderField(HTTPHeaderName::XContentTypeOptions)) != ContentTypeOptionsDisposition::Nosniff || equalLettersIgnoringASCIICase(responseMIMEType(), "text/css"_s);
 }
 
-bool CachedCSSStyleSheet::canUseSheet(MIMETypeCheckHint mimeTypeCheckHint, bool* hasValidMIMEType) const
+bool CachedCSSStyleSheet::canUseSheet(MIMETypeCheckHint mimeTypeCheckHint, bool* hasValidMIMEType, bool* hasHTTPStatusOK) const
 {
     if (errorOccurred())
         return false;
 
+    // https://html.spec.whatwg.org/#fetching-and-processing-a-resource-from-a-link-element:processresponseconsumebody
+    if (response().isNull() && !(response().httpStatusCode() >= 200 && response().httpStatusCode() <= 299)) {
```

Comment:
You can call `!response.isSuccessful()` to achieve the same thing.

Reply:
Thanks — made it so